system/i18n/cms_de.properties

```diff
@@ -768,3 +768,8 @@ urlRedirects.ruleNotFound.error=Die Regel wurde nicht gefunden
 urlRedirects.editRule.pageTitle=Regel bearbeiten
 urlRedirects.editRule.pageSubtitle={1}
 urlRedirects.editRule.breadcrumbTitle=Regel bearbeiten
+
```

Comment:
Please leave translations to our translators to save yourself effort. This needs to go through our translation workflow in Zanata.